src/schema/publication.graphql

```diff
@@ -50,6 +50,11 @@ type PublicationWithGene implements BasePublication {
   authors: [Author!]!
 }
 
+type NumberOfPublicationsWithGene {
+  num_pubs: Int!
+  publications: [PublicationWithGene]
```

Comment:
@JSneak I believe publications field should be required. I would change the type to `[PublicationWithGene!]!` since we know that the publications field is required and that the items are also required.